### ISSUE TITLE
Enable Kokkos profiling cmake options by default in ATDM configurations

### ISSUE THREAD

**Issue by jhux2** [trilinos/Trilinos#4190]
I would like to request that the two options
```
-D Kokkos_ENABLE_Profiling=ON
-D Teuchos_KOKKOS_PROFILING:BOOL=ON
```
be added to the various ATDM cmake configurations.  These are required to get useful information from the Kokkos profiling tools.  These are currently set to `OFF` by the ATDM configuration script (at least on waterman).

This means that after I do
```
source $TRILINOS_DIR/cmake/std/atdm/load-env.sh
```
I also have to remember to enable both options in my configure script.

From what I understand, having the two options `ON` by default will have negligible impact on performance *unless* you explicitly start profiling using the Kokkos tools.

@trilinos/framework 
@srajama1

**Comment by mhoemmen** [comment]
These options should be enabled by default — why does ATDM turn them off?

**Comment by rppawlo** [comment]
Last week we cleaned up the interaction between the StackedTimer and kokkos profiling (#4119). StackedTimers and TimeMonitors all feed into kokkos profiling. All panzer, EMPIRE and Drekar tests were passing with profiling on. So this should hopefully be a simple configure change.

**Comment by bartlettroscoe** [comment]
Is this issue not a duplicate of #4148?  Can we close this issue as a duplicate and communicate in #4148?

**Comment by bartlettroscoe** [comment]
Closed as duplicate of #4148.
